scripts/generate-tests.sh

```diff
@@ -273,7 +273,7 @@ tools/generate-raddstoreexpminusmax-test.py --spec test/f16-raddstoreexpminusmax
 tools/generate-raddstoreexpminusmax-test.py --spec test/f32-raddstoreexpminusmax.yaml --output test/f32-raddstoreexpminusmax.cc &
 
 ### Tests for VScaleExtExp micro-kernels
-tools/generate-vscaleextexp-test.py --spec test/f32-vscaleextexp.yaml --output test/f32-vscaleextexp.cc &
+tools/generate-vscaleextexp-test.py --tester VScaleExtExpMicrokernelTester --ukernel f32-vscaleextexp --output test/f32-vscaleextexp.cc &
```

Comment:
There's only one test using this, and now that it doesn't actually generate much code, we should just remove it. Let's also move the contents of vscaleextexp-microkernel-test.* to test/f32-vscaleextexp.cc as well.